errors.go

```diff
@@ -265,11 +267,11 @@ func ErrorToRFC6749Error(err error) *RFC6749Error {
 }
 
 type RFC6749Error struct {
-	Name        string `json:"error"`
-	Description string `json:"error_description"`
-	Hint        string `json:"error_hint,omitempty"`
-	Code        int    `json:"status_code,omitempty"`
-	Debug       string `json:"error_debug,omitempty"`
```

Comment:
If you want to make it explicit that these are not marshalled, you could add:

`json:“-“`

Reply:
I do not care about this message being marshaled. If somebody wants to do it for some other reasons (like storing in the database), I think they should be able. But it is not the structure we expose externally. In fact, I think I should return meaningful JSON names just for this case.

Reply:
Ok, SGTM